packages/web-twig/src/Resources/components/TextFieldBase/TextFieldBase.twig

```diff
@@ -53,7 +53,7 @@
 {%- set _styleProps = useStyleProps(props) -%}
 {%- set _rootClassNames = [ _rootClassName, _rootDisabledClassName, _rootFluidClassName, _rootValidationStateClassName, _styleProps.className ] -%}
 {%- set _labelClassNames = [ _labelClassName, _labelHiddenClassName, _labelRequiredClassName ] -%}
-{%- set _mainPropsWithoutReservedAttributes = props | filter((value, prop) => prop != 'id' and (_isAutoResizing and prop != 'data-toggle')) -%}
+{%- set _mainPropsWithoutReservedAttributes = props | filter((value, prop) => prop != 'id' and (_dataToggleAttr is null or prop != 'data-toggle')) -%}
```

Comment:
‼️ From the version [1.31.0](https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/CHANGELOG.md#features-1) it isn't possible to pass another props (such as data attributes) to the TextFieldBase components because of this condition. 😬 🙈 

Alternatively it's possible to write this logical expression as:
```
not (_dataToggleAttr is not null and prop == 'data-toggle')
```
but considering the readability of the code, I rewrote it to the applied formula.
Had to brush up my knowledge of mathematics and use [De Morgan's law](https://en.wikipedia.org/wiki/De_Morgan%27s_laws#Formal_notation)) but choose variant you prefer more. 😊